core/dbt/clients/agate_helper.py

```diff
@@ -41,4 +42,7 @@ def as_matrix(table):
 
 
 def from_csv(abspath):
-    return agate.Table.from_csv(abspath, column_types=DEFAULT_TYPE_TESTER)
+    with dbt.compat.open_file(abspath) as fp:
+        if fp.read(1) != u'\ufeff':
```

Comment:
did you try using `codecs.BOM_UTF8` instead of the hardcoded value here?

https://docs.python.org/2/library/codecs.html#codecs.BOM_UTF8
https://docs.python.org/3/library/codecs.html#codecs.BOM_UTF8

Reply:
I'm explicitly opening the value as unicode (that's what `open_file` does), so examining the unicode code point rather than the byte sequence is correct.

Reply:
Not suggesting that your comparison is wrong, just that you could replace the magic value `u'\ufeff'` with `codecs.BOM_UTF8.decode('utf-8')`